stravalib/client.py

```diff
@@ -1819,7 +1823,7 @@ def handle_subscription_callback(
         self,
         raw: dict[str, Any],
         verify_token: str = model.Subscription.VERIFY_TOKEN_DEFAULT,
-    ) -> dict[str, str]:
+    ) -> dict[str, Optional[str]]:
```

Comment:
The optional is not needed here better to only have str as in the docstring since it will always return a string. To silence the warning we can use an assert like this 
```py
assert callback.hub_challenge is not None
```

Reply:
that worked - thank you!